cmd/pullrequest-init/github_test.go

```diff
@@ -20,9 +20,10 @@ import (
 func TestGitHubParseURL(t *testing.T) {
 	wantOwner := "owner"
 	wantRepo := "repo"
+	wantHost := []string{"https://github.com", "https://github.com", "https://github.com", "http://github.com", "ssh://github.com", "https://example.com", "https://github.com",}    
```

Comment:
Keeping 2 separate arrays makes it unclear where how belongs to which URL. Instead, change the `url := range []string` below into:

```
tests := []struct {
  url string
  wantHost string
} {
 ...
}
for _, tc := range tests {
  ...
} 
```

See https://github.com/golang/go/wiki/TableDrivenTests for more details.

Reply:
Yes, it's better.  I update it.